engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-environment-modal.html

```diff
@@ -63,6 +58,10 @@ <h4 translate>Assign Lifecycle Environment and Content View</h4>
   </div>
 
   <div data-block="modal-footer">
+    <p style="text-align:left;font-size:12px;"><b>Note:</b> Updating the content view here will not automatically update the client. </p>
```

Comment:
I think the message is a little bit unclear. It almost makes it sound as if this is required, when it's really only required if you want it to take effect immediately.

What about "Changing the Content View will not affect the host until its next checkin. To update the host immediately run the following commands"


cc @sthirugn

Reply:
@jturel 
- I think you may be right, but I don't have time to validate this.  Are you sure about this or do we need to validate this?
- Assuming the above is validated, I like your messaging, maybe just capitalize `Host`

Reply:
Fixed and updated the wording.